src/gil.rs

```diff
@@ -52,10 +52,12 @@ pub(crate) fn gil_is_acquired() -> bool {
 /// software). Support for this is tracked on the
 /// [PyPy issue tracker](https://foss.heptapod.net/pypy/pypy/-/issues/3286).
 ///
+/// Python 3.6 only: If the Python interpreter is initialized but Python threading is not, this
```

Comment:
Is it only for python3.6? I can't see that python36 is different from the previous versions from the FFI doc. But I haven't tested on python35 or earlier though.

Reply:
We don't support Python 3.5 or earlier, as their end of life date has already been reached.

Note also that we will stop supporting Python 3.6 next year: https://github.com/PyO3/pyo3/issues/1782

Reply:
Got it. Thank you!